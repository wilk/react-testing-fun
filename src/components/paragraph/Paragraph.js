import React from 'react'

export default () => {
  return (
    <p>A paragraph</p>
  )
}