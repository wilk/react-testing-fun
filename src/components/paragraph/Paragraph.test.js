import React from 'react';
import { shallow } from 'enzyme';
import Paragraph from './Paragraph';

it('renders Paragraph', () => {
  const wrapper = shallow(<Paragraph />);
  expect(wrapper).toMatchSnapshot();
});