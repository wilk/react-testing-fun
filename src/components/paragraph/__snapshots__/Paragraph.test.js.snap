// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Paragraph 1`] = `
<p>
  A paragraph
</p>
`;
